### PR TITLE
feat: update register to have your say project link

### PR DESCRIPTION
**Pull request by cprieststephens**
## Describe your changes

2311: https://pins-ds.atlassian.net/jira/software/c/projects/ASB/boards/166?selectedIssue=ASB-2311
Update the project link on the register to have your say confirmation page.

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x] I have provided a description of the ticket
- [x] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: packages/forms-web-app/src/pages/projects/register/_common/complete/controller.js [hunk 1]
```diff
@@ -18,7 +19,7 @@ const getRegisterCompleteController = (req, res) => {
 			...viewModel[key],
 			ipRefNo,
 			email,
-			nsipProjectLink: nsipProjectLink(req.session.appData)
```
Comment (christopher-semmens) [inline packages/forms-web-app/src/pages/projects/register/_common/complete/controller.js:21]: Having a quick glance of the codebase - it looks like this is the only place the 'nsipProjectLink' function is being called. Might be a good opportunity to raise a tech debt ticket to investigate if it is redundant and if so, to remove it.

File: packages/forms-web-app/src/pages/projects/register/_common/complete/controller.js [hunk 2]
```diff
@@ -10,6 +10,7 @@ const {
 const getRegisterCompleteController = (req, res) => {
 	try {
 		const { session } = req;
+		const { caseRef } = session;
```
Comment (christopher-semmens) [inline packages/forms-web-app/src/pages/projects/register/_common/complete/controller.js:13]: I would be inclined to retrieve the case reference from the params. If we had a case where another controller did not have the caseRef added to the locals we would need to retrieve it from the params so following that rule should help to keep the codebase more consistent.

```suggestion
		const { params } = req;
		const { case_ref } = params;
```

File: packages/forms-web-app/src/pages/projects/register/_common/complete/controller.js [hunk 3]
```diff
@@ -18,7 +19,7 @@ const getRegisterCompleteController = (req, res) => {
 			...viewModel[key],
 			ipRefNo,
 			email,
-			nsipProjectLink: nsipProjectLink(req.session.appData)
+			projectURL: getProjectsURL(caseRef)
```
Comment (christopher-semmens) [inline packages/forms-web-app/src/pages/projects/register/_common/complete/controller.js:22]: This should use the getProjectsIndexURL function.

```suggestion
			projectsIndexURL: getProjectsIndexURL(caseRef)
```

File: packages/forms-web-app/src/pages/projects/register/_common/complete/controller.js (hunk 1 above)
Comment (cprieststephens) [reply packages/forms-web-app/src/pages/projects/register/_common/complete/controller.js:21]: Thanks, will do.

**Review by christopher-semmens — APPROVED** [review]

**Pull request merged**